src/atoms/cc-input-number.js

```diff
@@ -151,7 +166,12 @@ export class CcInputNumber extends LitElement {
     return html`
 
       ${this.label != null ? html`
-        <label for=${this._uniqueName}>${this.label}</label>
+        <label for=${this._uniqueInputId}>
+          ${this.label}
```

Comment:
Because `<label>` is a flex container, it's better for its direct children to be DOM elements and not text nodes. For this, I think `${this.label}` should be inside a span `<span>${this.label}</span>`